### PR TITLE
Add guide on what rate limit events are

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- New guide on what customer can expect when they are rate limited
- This is a request from Support so that they can direct customer to these docs from the rate limit UI events

### Merge instructions

Pending review, do not merge

### CODE REVIEW

File: content/en/metrics/guide/rate-limit.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- link: "/metrics/guide/custom_metrics_governance/"
+  tag: "Documentation"
+  text: "Best Practices for Custom Metrics Governance"
+---
+
+## Overview 
+
+Submitting metrics to Datadog with large numbers of unique tag values on a given key can result in high [cardinality][1]. Most often, this is caused by unbounded tags.
+
+Unbounded and high cardinality tags can impact your account’s performance and stability. To protect your account, Datadog monitors metric increases and notifies you when these submissions are rate-limited.
```
Comment (estherk15) [inline content/en/metrics/guide/rate-limit.md:17]:
```suggestion
Unbounded and high cardinality tags can impact your account's performance and stability. To protect your account, Datadog monitors metric increases and notifies you when these submissions are rate-limited.
```

File: content/en/metrics/guide/rate-limit.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+<div class="alert alert-warning">Only a Datadog Admin can request a removal of a metric rate limit. If you are not an Admin, make sure to include an Admin on the support ticket so they can confirm the request.</div>
+
+After making the changes to remove the unbounded tags, submit a request to [Datadog Support][5] to remove the rate limit. In your request, provide the following information: 
+- Name of the metric that was being rate limited 
+- Link to the rate limit event in the Event Platform
+- Confirmation a configuration change was done to remove the unbounded tags. 
+
+
+## Further reading
```
Comment (estherk15) [inline content/en/metrics/guide/rate-limit.md:62]:
```suggestion
## Further Reading
```

**Comment by buraizu** [comment]
Thanks, created DOCS-9028 to review

File: content/en/metrics/guide/rate-limit.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+For more information, see the [Best Practices for Custom Metrics Governance][4] guide.
+
+## Submit a request to remove the rate limit
+
+<div class="alert alert-warning">Only a Datadog Admin can request a removal of a metric rate limit. If you are not an Admin, make sure to include an Admin on the support ticket so they can confirm the request.</div>
+
+After making the changes to remove the unbounded tags, submit a request to [Datadog Support][5] to remove the rate limit. In your request, provide the following information: 
+- Name of the metric that was being rate limited 
+- Link to the rate limit event in the Event Platform
+- Confirmation a configuration change was done to remove the unbounded tags.
```
Comment (michaelcretzman) [inline content/en/metrics/guide/rate-limit.md:59]:
```suggestion
- Name of the rate-limited metric 
- Link to the rate limit event in the Event Platform
- Confirmation that a configuration change removed the unbounded tags
```

**Review by michaelcretzman — APPROVED** [review]
Approved with minor suggestions

**Pull request merged**